### ISSUE TITLE
err 5xx when posting a reply

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#692]
strange behavior after posting a reply on the alpha (version 0.6.0)

POST https://alpha.caliopen.org/api/v1/messages

```
body | foo
discussion_id | 7b6e6c61-ef93-4473-a147-335379838095
identities | {…}
parent_id | cf53ed35-2f3f-449b-8400-a6e8a227163b
subject | 1
```

response

```
{
	"errors": [{
		"code": 520,
		"description": "",
		"component": "collection_message",
		"values": 520,
		"message": "vars() argument must have __dict__ attribute",
		"property": "MergePatchError",
		"type": "Unknown Error"
	}]
}
```

**Comment by sapiens-sapide** [comment]
Difficult to reproduce, but may be fixed by #836ba86.

**Comment by gdchamal** [comment]
Sound fixed, no error like that since last deploy
